components/analytics-dashboard/org.wso2.carbon.analytics.dashboard.admin/src/main/java/org/wso2/carbon/analytics/dashboard/admin/data/Dashboard.java

```diff
@@ -15,14 +15,14 @@
  */
 package org.wso2.carbon.analytics.dashboard.admin.data;
 
-import org.apache.axis2.AxisFault;
+import org.wso2.carbon.analytics.dashboard.admin.exception.InvalidRequestException;
```

Comment:
What is the intended purpose of this exception class? Does is inform the user about malformed request? Or something wrong happened while processing the message?